app/routes/event-projection-detail.js

```diff
@@ -19,7 +19,7 @@ module.exports = [{
 
       if (blobData?.blobList.length) {
         projectionData = await getProjection(`${blobData.blobList[0].blob}`)
-        eventDetails = eventType ? projectionData?.events.find(x => x.eventType === eventType) : projectionData.events.find(x => x.eventType === 'batch-processing')
+        eventDetails = eventType ? projectionData?.events.find(x => x.eventType === eventType) : projectionData.events.find(x => x.eventType === 'payment-request-enrichment')
```

Comment:
This event does not exist if payments do not originate from Siti Agri.  Updating to first event that applies to all requests.